app/views/admin/auctions/show.html.erb

```diff
@@ -19,13 +19,13 @@
 <p><%=h raw @auction.html_description %></p>
 
 <div class='usa-grid'>
-  <% if @auction.current_bid? %>
+  <% if @auction.highlighted_bid? %>
```

Comment:
Null object? Keeping `if` statements out of the view is pretty important. You can move this to the presenter/view model by creating a null partial to handle the false case.